src/components/Example1.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import '../styles/example1.css';
+import { addData, getData } from '../utils/indexedDB';
+
+interface Certificate {
```

Comment:
You already have exactly same certificate interface defined in data.ts, so please use that one. 
In general, try to not duplicate code.